src/components/SDCBadge.tsx

```diff
@@ -10,7 +10,7 @@ type SDCBadgeProps = BadgeProps & { isOff?: boolean };
 export const SDCBadge: React.FC<SDCBadgeProps> = ({ isOff, ...restProps }) => {
     return (
         <Badge
-            borderRadius="full"
+            borderRadius="6px"
```

Comment:
We actually want our badges to be round, like in the design!

```suggestion
            borderRadius="full"
```